kolibri/core/assets/src/views/AppError/ReportErrorModal.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+
+  <KModal
+    :title="$tr('reportErrorHeader')"
+    :cancelText="$tr('closeErrorModalButtomPrompt')"
+    class="error-detail-modal"
+    size="large"
+    @cancel="$emit('cancel')"
+  >
+
+    <section>
+      <h3 v-if="offline">
+        {{ $tr('forumPrompt') }}
+      </h3>
+      <p> {{ $tr('forumUseTips') }} </p>
+      <p> {{ $tr('forumPostingTips') }} </p>
+      <KExternalLink
+        class="download-as-text-link"
+        :text="forumLink"
+        :href="forumLink"
+      />
+    </section>
+
+    <!-- only when offline -->
+    <section v-if="offline">
+      <h3> {{ $tr('emailPrompt') }} </h3>
+      <p> {{ $tr('emailDescription') }} </p>
+      <!-- email link goes here. TODO Probably not an href? -->
+      <KExternalLink
+        :text="emailAddress"
+        :href="emailAddressLink"
+      />
+    </section>
+
+    <section>
+      <h3>
+        {{ $tr('errorDetailsHeader') }}
+      </h3>
+      <pre ref="errorLog" class="error-log">{{ error }}</pre>
+    </section>
+
+
+    <section class="error-copying-options">
+      <div>
+        <KButton
+          v-if="clipboardCapable"
+          class="copy-to-clipboard-button"
+          :primary="false"
+          :text="$tr('copyToClipboardButtonPrompt')"
+          ref="errorCopyButton"
+        />
+      </div>
+      <div>
+        <KExternalLink
+          :text="$tr('downloadAsTextPrompt')"
+          :download="errorTextFileName"
+          :href="errorTextFileLink"
+        />
+      </div>
+    </section>
+
+
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import { mapState, mapActions } from 'vuex';
+  import KButton from 'kolibri.coreVue.components.KButton';
+  import KExternalLink from 'kolibri.coreVue.components.KExternalLink';
+  import KModal from 'kolibri.coreVue.components.KModal';
+  import ClipboardJS from 'clipboard';
+
+  export default {
+    name: 'ReportErrorModal',
+    $trs: {
+      reportErrorHeader: 'Report Error',
+      forumPrompt: 'Visit the community forums',
+      // reall long
+      forumUseTips:
+        'Search the community forum to see if others encountered similar issues. If unable to find anything, paste the error details below into a new forum post so we can rectify the error in a future version of Kolibri.',
+      forumPostingTips:
+        'Include a description of what you were trying to do and what you clicked on when the error appeared.',
+      emailPrompt: 'Send an email to the developers',
+      emailDescription:
+        'Contact the support team with your error details and we’ll do our best to help.',
+      errorDetailsHeader: 'Error details',
+      copyToClipboardButtonPrompt: 'Copy to clipboard',
+      copiedToClipboardConfirmation: 'Copied to clipboard',
+      downloadAsTextPrompt: 'Or download as .txt file',
```

Comment:
`as a .txt file`?

Reply:
Weird discrepancy between screencap and doc strings @khangmach 
https://docs.google.com/document/d/1X-p6Vnj3aSnQFC8lF_ZZo7mIvnMAwNZBbruPbL3FrOc/edit

Reply:
Looks like I forgot the "a". Christian's suggestion works too.